### ISSUE TITLE
OmniSharp server load timed out

### ISSUE THREAD

**Issue by mrsdevo** [dotnet/vscode-csharp#4553]
Issue Type: <b>Bug</b>

 

        {

            "name": ".NET Core Attach",

            "type": "coreclr",

            "request": "attach",

            "processId": "${command:pickProcess}"

        },

Error:

Couldnt find a debug adapter descriptor for debug type coreclr(extension might have failed to activate)

 

Extension version: 1.23.11

VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:35:57.887Z)

OS version: Windows_NT x64 10.0.19042

**Comment by gregg-miskelly** [comment]
@mrsdevo Please try uninstalling and reinstalling the C# extension. If that doesn't work, what do you see in you switch to the 'Output' tab, and look at the 'OmniSharp Log' pane.

**Comment by mrsdevo** [comment]
Tried reinstalling , didn't help.
In Omnisharp log, I see this
[ERROR] Error: OmniSharp server load timed out. Use the 'omnisharp.projectLoadTimeout' setting to override the default delay (one minute).

I changed that to 2 mins, but still see the error.

**Comment by keocra** [comment]
TLDR;: Try going into the extension settings and disable "Pass the --debug flag when launching..." instead of changing the 'omnisharp.projectLoadTimeout' setting. 

I had the same issue. OmniSharp server not starting. Always having the error "...timed out...". In the end it turned out, that passing the --debug flag fails the server to start and increasing the time for the project to load were not changing anything.

**Comment by JoeRobich** [comment]
A lot has changed with O# project loading since this issue was opened. If you are still having problems, please open a new issue at https://github.com/omnisharp/omnisharp-roslyn/issues
